### PR TITLE
Miscellaneous documentation and bug fixes

### PR DESCRIPTION
**Pull request by 80degreeswest**
Fix few minor issues from the review comments.
Referenced new config.yml file in all examples.
Updated existing examples with new yml configs.
Removed line numbers from docs as those are bound to get outdated with future configs changes.

### CODE REVIEW

File: _site/docs/architecture/content_addressable_storage.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Since these implementations vary in complexity and storage semantics, a common i
 
 The memory CAS implementation is extremely simple in design, constituting a maximum size with LRU eviction policy. Entry eviction is a registrable event for use as a storage for the delegated ActionCache, and Writes may be completed asynchronously by concurrent independent upload completion of an entry.
 
-This is the example presentation of a CAS in the memory instance available [here](https://github.com/bazelbuild/bazel-buildfarm/blob/85c3fdaf89fedc2faee1172fab01338777de79a1/examples/config.memory.yml#L40-L45), but for reference, specification in any `cas_config` field for server or worker will enable the creation of a unique instance.
+This is the example presentation of a CAS in the memory instance available [here](https://github.com/bazelbuild/bazel-buildfarm/blob/main/examples/config.memory.yml#L40-L45), but for reference, specification in any `cas_config` field for server or worker will enable the creation of a unique instance.
```
Comment (jacobmou) [inline _site/docs/architecture/content_addressable_storage.md:37]:
```suggestion
This is the example presentation of a CAS in the memory instance available [here](https://github.com/bazelbuild/bazel-buildfarm/blob/main/examples/config.memory.yml#L5-L6), but for reference, specification in any `cas_config` field for server or worker will enable the creation of a unique instance.
```

File: _site/docs/architecture/content_addressable_storage.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ cas_config: {
 
 This is a CAS which completely mirrors a target CAS for all requests, useful as a proxy to be embedded in a full Instance declaration.
 
-A grpc config example is available in the alternate instance specification in the memory server example [here](https://github.com/bazelbuild/bazel-buildfarm/blob/85c3fdaf89fedc2faee1172fab01338777de79a1/examples/config.memory.yml#L91-L99). For reference:
+A grpc config example is available in the alternate instance specification in the memory server example [here](https://github.com/bazelbuild/bazel-buildfarm/blob/main/examples/config.memory.yml#L91-L99). For reference:
```
Comment (jacobmou) [inline _site/docs/architecture/content_addressable_storage.md:52]: The line number also need to be updated.

File: _site/docs/architecture/content_addressable_storage.md [hunk 3]
```diff
@@ -77,7 +77,7 @@ And will result in a listening grpc service on port 8081 on all interfaces, rela
...
 
-A shard CAS is the default for the Shard Instance type, with its required [backplane specification](https://github.com/bazelbuild/bazel-buildfarm/blob/85c3fdaf89fedc2faee1172fab01338777de79a1/examples/config.shard.yml#L17-L239). Since functionality between Shard CAS, AC, and Execution are mixed in here, the definition is somewhat cluttered, with efforts to refine specific aspects of it underway.
+A shard CAS is the default for the Shard Instance type, with its required [backplane specification](https://github.com/bazelbuild/bazel-buildfarm/blob/main/examples/config.shard.yml#L17-L239). Since functionality between Shard CAS, AC, and Execution are mixed in here, the definition is somewhat cluttered, with efforts to refine specific aspects of it underway.
```
Comment (jacobmou) [inline _site/docs/architecture/content_addressable_storage.md:80]: line numbers.

File: _site/docs/architecture/content_addressable_storage.md [hunk 4]
```diff
@@ -88,7 +88,7 @@ The worker's CAS file cache uses persistent disk storage. A strongly recommended
...
+Each Worker type's specification of the CASFileCache is unique. The memory worker defaults to CASFileCache with no option to change it, only configure its [root directory](https://github.com/bazelbuild/bazel-buildfarm/blob/main/examples/config.memory.yml#L46-L47) and [configuration options](https://github.com/bazelbuild/bazel-buildfarm/blob/85c3fdaf89fedc2faee1172fab01338777de79a1/examples/config.memory.yml#L111-L117). The shard worker allows a more flexible specification, with delegates available of the other types to fall back on for CAS expansion, and [encapsulated CAS configs](https://github.com/bazelbuild/bazel-buildfarm/blob/85c3fdaf89fedc2faee1172fab01338777de79a1/examples/config.shard.yml#L49-L65)
```
Comment (jacobmou) [inline _site/docs/architecture/content_addressable_storage.md:91]: line numbers.

File: _site/docs/architecture/queues.md [hunk 5]
```diff
@@ -19,7 +19,7 @@ Some operations' Actions may have specific platform requirements in order to exe
...
+Provision queues are intended to represent particular operations that should only be processed by particular workers. An example use case for this would be to have two dedicated provision queues for CPU and GPU operations. CPU/GPU requirements would be determined through the [remote api's command platform properties](https://github.com/bazelbuild/remote-apis/blob/main/build/bazel/remote/execution/v2/remote_execution.proto#L595). We designate provision queues to have a set of "required provisions" (which match the platform properties). This allows the scheduler to distribute operations by their properties and allows workers to dequeue from particular queues.
```
Comment (jacobmou) [inline _site/docs/architecture/queues.md:22]: line numbers

**Review by jacobmou — COMMENTED** [review]
Thanks for updating docs! 
Seems like the line numbers referenced in these docs also need to be updated before we want to land it.

File: _site/docs/architecture/content_addressable_storage.md (hunk 1 above)
Comment (80degreeswest) [reply _site/docs/architecture/content_addressable_storage.md:37]: Oh yeah good. catch. I'll update.

**Review by jacobmou — APPROVED** [review]

File: _site/docs/architecture/queues.md [hunk 6]
```diff
@@ -56,56 +56,44 @@ Otherwise, the execution request's properties must have exactly the same set of
...
+        - name: "max-cores"
+          value: "*"
+    - name: "gpu"
+      allowUnmatched: true
+      properties:
+        - name: "gpu"
+          value: "1"
 ```
 
-In addition to the queue declaration, the worker must specify from which queue tasks are obtained (otherwise the worker process will refuse to start):
```
Comment (cameron-martin) [inline _site/docs/architecture/queues.md:98]: Is there going to be a replacement for this functionality when v2 is released?

Comment (80degreeswest) [reply _site/docs/architecture/queues.md:98]: That was an incorrect documentation. The platform in dequeue match settings was actually not being used hence it was removed. The queue definitions are sufficient to match worker to a pool of workers.

**Pull request merged**